boss-ws/src/main/java/org/genomebridge/boss/http/models/ObjectCore.java

```diff
@@ -11,10 +16,16 @@ public void copy( ObjectCore that ) {
         this.ownerId = that.ownerId;
     }
 
+    @ApiModelProperty(value = "string containing the ID assigned by BOSS when the object was created")
     public String objectId;
+    @ApiModelProperty(value = "string assigned by the caller that is a display name for this object")
     public String objectName;
+    @ApiModelProperty(value = "specifies the storage platform containing the underlying data object that BOSS is referencing")
     public String storagePlatform;
+    @ApiModelProperty(value = "string that should be a valid URI. This field is only present for “opaqueURI” objects")
     public String directoryPath;
+    @ApiModelProperty(value = "a number representing an estimate of the size of the underlying data object")
     public Long sizeEstimateBytes;
+    @ApiModelProperty(value = "the name of the user who is the “owner” of the data")
```

Comment:
Nitpick detail - I don't think the left/right quotes here are utf-8. Can we play it safe with single quotes? This could be a non-issue in git, but I've run into trouble with these kinds of characters in older source control systems.